### DISCUSSION TITLE
Sending a signal

### DISCUSSION THREAD

**Discussion by amirmakramkolta** [elsa-workflows/elsa-core#2644] [Q&A]
Hi,
I tried the method of Sending signal as in [docs ](https://elsa-workflows.github.io/elsa-core/docs/guides/guides-workflow-contexts) using  API endpoint like this "http://localhost:5001/v1/signals/send-message/execute"  with Content-Type: "application/json" and workflowInstanceId in the body but I get received this response:
`
{
    "$id": "1",
    "startedWorkflows": []
}
` 
and workflow instance status is still Suspended

**Comment by sfmskywalker** [comment]
Hi Amir,

I just tried it myself, but it worked well for me.
However, I did try with the latest 2.4.1-preview.17 packages available from MyGet. Perhaps you can do the same. This is especially important since the dashboard docker image is also built against the latest source code from the master branch, which means it uses client-side code that is incompatible with older versions than Elsa 2.4.

If you're still stuck after trying the above, then feel free to send me your solution + steps to reproduce and I will take a look.
